security/certificate_types_descriptions/machine-config-operator-certificates.adoc

```diff
@@ -8,24 +8,48 @@ toc::[]
 
 == Purpose
 
-Machine Config Operator certificates are used to secure connections between the Red Hat Enterprise Linux CoreOS (RHCOS) nodes and the Machine Config Server.
+This certificate authority is used to secure connections from nodes to Machine Config Server (MCS) during initial provisioning.
+
+There are two certificates:
+. A self-signed CA, the MCS CA
+. A derived certificate, the MCS cert
+
+=== Provisioning details
+
+{product-title} installations using {op-system-first} are installed using Ignition. This process is split into two parts:
+
+. An Ignition config is created which is references a URL for the full configuration served by the MCS.
```

Comment:
```suggestion
. An Ignition config is created that references a URL for the full configuration that is served by the MCS.
```
Using "that" instead of "which", as I think this sentence contains a restrictive clause. Source: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-with-nonrestrictive-clauses-and-phrases